src/core/jsproxy.c

```diff
@@ -445,6 +456,74 @@ JsProxy_Bool(PyObject* o)
   return hiwire_get_bool(self->js) ? 1 : 0;
 }
 
+#define QUIT_IF_NULL(x)                                                        \
+  do {                                                                         \
+    if (x == NULL) {                                                           \
+      goto finally;                                                            \
+    }                                                                          \
+  } while (0)
```

Comment:
This should already define in the headers?

Reply:
Oops, yes rather recently =)